### PR TITLE
Add infrastructure for Tasks TCK

### PR DESCRIPTION
**Pull request by ericbottard**
Other tests to be added later

Fixes spring-cloud/spring-cloud-deployer#104

### CODE REVIEW

File: spring-cloud-deployer-spi-test/src/main/java/org/springframework/cloud/deployer/spi/test/AbstractTaskLauncherIntegrationTests.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+
+
+
+	/**
+	 * Return a resource corresponding to the spring-cloud-deployer-spi-test-app app suitable for the target runtime.
+	 *
+	 * The default implementation returns an uber-jar fetched via Maven. Subclasses may override.
+	 */
+	protected Resource integrationTestProcessor() {
```
Comment (ericbottard) [inline spring-cloud-deployer-spi-test/src/main/java/org/springframework/cloud/deployer/spi/test/AbstractTaskLauncherIntegrationTests.java]: That will need to be replaced by a task-specific equivalent.

Also, ideally the app-specific artifact should change name (as would this method), but this is a bit disruptive.
Hopefully `@Override` helps here..

**Comment by mminella** [comment]
Was the branch for this PR deleted?  I can't seem to find it either when I fetch from @ericbottard repo or when I browse it via Github...

**Comment by ericbottard** [comment]
You scared me @mminella . It's here: https://github.com/ericbottard/spring-cloud-deployer/tree/issue_104_TCK_tasks

**Comment by mminella** [comment]
LGTM.  Merged as 7294950f3470132a7d23fcf31c49477bc47065ae 

P.S.  Sorry about the scare.  I was looking in the cf deployer project :(

**Pull request closed without merging**